go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/grpc-ecosystem/grpc-gateway/v2 v2.15.0
 	github.com/ipfs/go-ipfs-api v0.3.0
 	github.com/lestrrat-go/jwx/v2 v2.0.8
-	github.com/medibloc/panacea-core/v2 v2.0.6-0.20230113040800-7e7a4728ab82
+	github.com/medibloc/panacea-core/v2 v2.0.6-0.20230202103417-de758e7f84e9
```

Comment:
This must be updated once https://github.com/medibloc/panacea-core/pull/627 is merged.

Reply:
I got it. 
I'll change this version once the panacea core PR merge.

Reply:
Done 3a4af56ac1e11005315723984b2baf9acbdb3e8d